Classes/Service/TwoFactorAuthenticationService.php

```diff
@@ -6,7 +6,7 @@
 use Doctrine\DBAL\Connection;
```

Comment:
The one line above can be removed now
```use Doctrine\Common\Persistence\ObjectManager as DoctrineObjectManager;```

Reply:
@albe Uh, which one? There is no `... as DoctrineObjectManager` line left?!

Reply:
Need to view whole file or expand to top. It is there ;)

Reply:
Doh, right. Done!